### PR TITLE
No shapes in profilings by default (#233)

### PR DESCRIPTION
**Pull request by astachowiczhabana**
1. Add option profiling_record_shapes in run_generation (default False) - reducing profiling files size
2. Remove option with_stack in profiler - reducing profiling files size

### CODE REVIEW

File: optimum/habana/transformers/training_args.py [hunk 1]
```diff
@@ -118,6 +118,8 @@ class GaudiTrainingArguments(TrainingArguments):
             Number of steps to ignore for profling.
         profiling_steps (`int`, *optional*, defaults to 0):
             Number of steps to be captured when enabling profiling.
+        profiling_record_shapes (`bool`, *optional*, defaults to True):
+            Record shapes when enabling profiling.
```
Comment (regisss) [inline optimum/habana/transformers/training_args.py:122]: I don't think we should add this as there is no new argument defined in `training_args.py.

**Review by regisss — COMMENTED** [review]
Please run `make style`

File: optimum/habana/transformers/training_args.py (hunk 1 above)
Comment (libinta) [reply optimum/habana/transformers/training_args.py:122]: where should we add this?

Comment (regisss) [reply optimum/habana/transformers/training_args.py:122]: IT's already added in the docstrings of `generation/utils.py` and in the help section of the new arg of the text-generation example. This is good enough.

Comment (ssarkar2) [reply optimum/habana/transformers/training_args.py]: removed

**Review by regisss — APPROVED** [review]

**Pull request merged**